CHANGES.rst

```diff
@@ -8,13 +8,54 @@ details, see the commit logs at https://github.com/scikit-build/scikit-build
 Next Release
 ============
 
-We are hard at work on the next generation of scikit-build
-`scikit-build-core <https://github.com/scikit-build/scikit-build-core>`_, which
-has had it's first non-development release. We are also continuing to fix bugs,
-make improvements, and backport changes here.
+We are hard at work on the next generation of scikit-build `scikit-build-core
+<https://github.com/scikit-build/scikit-build-core>`_, which will eventually
+replace the backend here. We are also continuing to fix bugs, make improvements,
+and backport changes here.
 
 .. START-BRIEF-CHANGELOG
 
+Scikit-build 0.17.0
+===================
+
+A lot of bug fixes are present in this release, focusing on Windows, PyPy, and
+cross compiling. We've also improved the compatibility with default setuptools
+behaviors a little, and enabled some things that were previously unavailable,
+like overriding the build type via the cmake argument environment variables.
+We've expanded our CI matrix to include Windows and macOS PyPy and some Fortran
+tests on Linux. This release requires Python 3.7+.
+
+Bug fixes
+---------
+
+* Match setuptools behavior for ``include_package_data`` default. by :user:`vyasr` in :pr:`873`
+* Misc. fixes for F2PY and PythonExtensions modules by :user:`benbovy` in :pr:`495`
+* Provide more useful error if user provides ``CMAKE_INSTALL_PREFIX`` by :user:`vyasr` in :pr:`872`
+* Stop assuming that ``.pyx`` files are in the same directory as ``CMakeLists.txt`` by :user:`vyasr` in :pr:`871`
+* Allow build type overriding by :pr:`902`
+* Detect PyPy library correctly on Windows (#701) in :pr:`904`
+* Include library for FindPython for better Windows cross-compiles in :pr:`913`
+* Use f2py's ``get_include`` if present by in :pr:`877`
+* Fix support for cross-compilation exception using ``targetLinkLibrariesWithDynamicLookup`` by :user:`erykoff` in :pr:`901`
```

Comment:
```suggestion
* Fix support for cross-compilation exception using ``targetLinkLibrariesWithDynamicLookup`` by :user:`erykoff` and :user:`jcfr` in :pr:`901`
```

Reply:
Same thing, I've generally avoided listing us. If you want to start (I don't care either way), they will all need updates.